ant-node-manager/Cargo.toml

```diff
@@ -34,7 +34,7 @@ ant-build-info = { path = "../ant-build-info", version = "0.1.29" }
 ant-evm = { path = "../ant-evm", version = "0.1.17" }
 ant-logging = { path = "../ant-logging", version = "0.3.0" }
 ant-protocol = { path = "../ant-protocol", version = "1.0.9" }
-ant-releases = { version = "0.4.1" }
+ant-releases = { version = "0.4.2" }
```

Comment:
just to confirm, so here doesn't need to use the same bespoken branch as ant-node ?